src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -65,18 +72,21 @@ def _parse_data(self, data: PandaPowerData, data_type: str, extra_info: Optional
         Returns:
             Converted power-grid-model data
         """
-
         # Clear pgm data
         self.pgm_input_data = {}
         self.idx_lookup = {}
         self.next_idx = 0
 
-        # Set pandas data
-        self.pp_input_data = data
+        self.pgm_update_data = {}
 
         # Convert
         if data_type == "input":
+            # Set pandas data
+            self.pp_input_data = data
             self._create_input_data()
+        elif data_type == "update":
+            self.pp_update_data = data
+            self._update_input_data()
```

Comment:
since this function is called `_parse_data`, this may not be the right place to actually `set` the data. could be out of scope though

Alternatively, maybe it's a good moment to split the (minor) chunks of differences between `input_data`, `output_data` and `update_data` into separate classes